COLLABORATOR_GUIDE.md

```diff
@@ -0,0 +1,96 @@
+# jade Collaborator Guide
```

Comment:
Can it be in `CONTRIBUTING.md` as well? It's not a great idea to have a bunch of UPPERCASE files in the root of the project.